samples/matter/common/src/app/migration_manager.cpp

```diff
@@ -0,0 +1,43 @@
+/*
+ * Copyright (c) 2024 Nordic Semiconductor ASA
+ *
+ * SPDX-License-Identifier: LicenseRef-Nordic-5-Clause
+ */
+
+#include "migration_manager.h"
+
+LOG_MODULE_DECLARE(app, CONFIG_CHIP_APP_LOG_LEVEL);
+
+namespace Nrf::Matter
+{
+namespace Migration
```

Comment:
I don't feel like `app` directory is the best place to put it. At least after talks with @markaj-nordic I understood we should keep the most common to all cases API for Matter and migration manager is something very specific that will not be used on daily basis. I would recommend creating some another directory, like `crypto`,`psa` or `migration`.